aspnetcore/razor-components/components.md

```diff
@@ -479,6 +479,32 @@ The [sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/razor-com
 
 The base class should derive from `ComponentBase`.
 
+## Importing components
+
+Components infer their namespace based on the file path and the file name.
```

Comment:
I would saw that the namespace of a component authored with Razor is based on:
- the project's `RootNamespace`
- the path from the project root to the component

That's a little more accurate.